docs/docs/server/introduction/accounts-and-projects.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Accounts and projects
+titleTemplate: :title | Introduction | Server | Tuist
+description: Learn how to create and manage accounts and projects in Tuist.
+---
+
+# Accounts and projects
+
+## Accounts
+
+To use the server, you'll need an account. There are two types of accounts:
+
+- **Personal account:** Those accounts are created automaticaly when you sign up and are identified by a handle that's obtained either from the identity provider (e.g. GitHub) or the first part of the email address.
+- **Organization account:** Those accounts are manually created and are identified by a handle that's defined by the developer. Organizations allow inviting other members to collaborate on projects.
+
+If you are familiar with [GitHub](https://github.com), the concept is similar to theirs, where you can have personal and organization accounts, and they are identified by a *handle* that's used when constructing URLs.
+
+> [!NOTE] CLI-FIRST
+> Most operations to manage accounts and projects are done through the CLI. We are working on a web interface that will make it easier to manage accounts and projects.
+
+You can manage the organization through the sub commands under `tuist organization`.
+
+## Projects
+
+Your projects, either Tuist's or raw Xcode's, need to be integrated with your account through a server-side project. Continuing with the comparison with GitHub, it's like having a local and a remote repository where you push your changes. You can use the `tuist project` to create and manage projects.
+
+Projects are identified by a full-handle, which is the result of concatenating the organization handle and the project handle. For example, if you have an organization with the handle `tuist`, and a project with the handle `tuist`, the full-handle would be `tuist/tuist`.
```

Comment:
```suggestion
Projects are identified by a full handle, which is the result of concatenating the organization handle and the project handle. For example, if you have an organization with the handle `tuist`, and a project with the handle `tuist`, the full-handle would be `tuist/tuist`.
```

Is there any specific reason why to use `full-handle` over `full handle`? There are a couple of extra mentions below if you decide to change that.

Reply:
Not really. I'll change it to full handle.